src/clients/stormGlass.ts

```diff
@@ -75,10 +77,30 @@ export class StormGlass {
     'swellDirection,swellHeight,swellPeriod,waveDirection,waveHeight,windDirection,windSpeed';
   readonly stormGlassAPISource = 'noaa';
 
-  constructor(protected request = new HTTPUtil.Request()) {}
+  constructor(
+    protected request = new HTTPUtil.Request(),
+    protected cacheUtil = new CacheUtil()
+  ) {}
 
-  //TODO someone to cache this call
   public async fetchPoints(lat: number, lng: number): Promise<ForecastPoint[]> {
+    const cacheForecastPoints = this.getForecastPointsFromCache(lat, lng);
+
+    if (!cacheForecastPoints) {
+      const forecastPoints = await this.getForecastPointsFromApi(lat, lng);
+      this.setForecastPointsInCache(
+        `forecast_points_${lat}_${lng}`,
+        forecastPoints
+      );
+      return forecastPoints;
+    }
+
+    return cacheForecastPoints;
+  }
+
+  public async getForecastPointsFromApi(
```

Comment:
any reason for this method to be public? I think it should be at least `protected`, there's no reason to expose it in my option.

Reply:
There's no reason, I'll change it!